lib/rules/no-unused-class.js

```diff
@@ -107,14 +109,14 @@ export default {
            ```
            then the loop will be run 2 times
          */
-        _.forOwn(map, (o) => {
+        _.forIn(map, (o) => {
```

Comment:
I didn't get the reason for using forIn instead of forOwn.
From the documentation
* forIn `Iterates over own and inherited enumerable string keyed properties of an object`
* forOwn `Iterates over own enumerable string keyed properties of an object`